README.md

```diff
@@ -14,12 +14,7 @@ in Bazel, and by the
 [`swift_library` rule](https://github.com/bazelbuild/rules_swift/blob/master/doc/rules.md#swift_library)
 available from [rules_swift](https://github.com/bazelbuild/rules_swift).
 
-These rules require Bazel 2.0.
-
-## Reference documentation
-
-[Click here](https://github.com/ob/rules_ios/tree/master/doc)
```

Comment:
should still link to the docs

Reply:
will point to `https://github.com/bazel-ios/rules_ios/tree/master/docs` same for the other deletion i made